src/graphql/schema.graphql

```diff
@@ -24,7 +30,43 @@ type Mutation {
 
     "Whether or not the contributor wants to receive email updates."
     subscribe: Boolean!
-  ): DiscountCode!
+  ): DiscountCode! @deprecated(reason: "use createContributor instead")
```

Comment:
TODO: make sure all deprecated methods still work before we ship this — can't break the existing store UI!